src/background/extension.ts

```diff
@@ -410,11 +410,15 @@ style && style.parentElement.removeChild(style);
          * installed in system.
          */
         protected getFontList(onReturned: (fonts: string[]) => void) {
-            chrome.fontSettings.getFontList((res) => {
-                // id or name?
-                var fonts = res.map((r) => r.fontId);
-                onReturned(fonts);
-            });
+            if(chrome.fontSettings != undefined){
+                chrome.fontSettings.getFontList(function (res) {
+                    // id or name?
+                    var fonts = res.map(function (r) { return r.fontId; });
+                    onReturned(fonts);
+                });
+            } else {
+                onReturned([]);
```

Comment:
Let's provide at least generic font names instead of empty list https://developer.mozilla.org/en/docs/Web/CSS/font-family#<generic-name>

Currently it is not possible to enter font name but it should be picked from list.